core/src/test/java/org/testcontainers/images/ParsedDockerfileTest.java

```diff
@@ -40,6 +46,12 @@ public void ignoringBuildStageNames() {
         assertEquals("ignores build stage names and allows multiple images to be extracted", Sets.newHashSet("someimage", "nextimage"), parsedDockerfile.getDependencyImageNames());
     }
 
+    @Test
+    public void ignoringPlatformArgs() {
+        final ParsedDockerfile parsedDockerfile = new ParsedDockerfile(asList("FROM --platform=linux/amd64 someimage", "RUN something"));
```

Comment:
since the parsing seem to support multiple flags, perhaps we should add this (2 or more flags) case too

Reply:
Platform seems to be the only arg that is supported by docker at present, but sure, seems like a good idea.